apps/nextjs-website/react-components/types/Video/Video.types.ts

```diff
@@ -0,0 +1,47 @@
+import { CommonProps, Generic } from '../common/Common.types';
+
+export interface VideoImageProps extends CommonProps {
```

Comment:
Either call this `VideoProps` or rename the component `VideoImage` for consistency.

I'd go for the latter.